README.rst

```diff
@@ -35,38 +35,48 @@ Error Codes
 
 These error codes are emitted:
 
-+------+-------------------------------------------------------+
-| code | sample message                                        |
-+======+=======================================================+
-| N801 | class names should use CapWords convention            |
-+------+-------------------------------------------------------+
-| N802 | function name should be lowercase                     |
-+------+-------------------------------------------------------+
-| N803 | argument name should be lowercase                     |
-+------+-------------------------------------------------------+
-| N804 | first argument of a classmethod should be named 'cls' |
-+------+-------------------------------------------------------+
-| N805 | first argument of a method should be named 'self'     |
-+------+-------------------------------------------------------+
-| N806 | variable in function should be lowercase              |
-+------+-------------------------------------------------------+
-| N807 | function name should not start and end with '__'      |
-+------+-------------------------------------------------------+
-+------+-------------------------------------------------------+
-| N811 | constant imported as non constant                     |
-+------+-------------------------------------------------------+
-| N812 | lowercase imported as non lowercase                   |
-+------+-------------------------------------------------------+
-| N813 | camelcase imported as lowercase                       |
-+------+-------------------------------------------------------+
-| N814 | camelcase imported as constant                        |
-+------+-------------------------------------------------------+
-| N815 | mixedCase variable in class scope                     |
-+------+-------------------------------------------------------+
-| N816 | mixedCase variable in global scope                    |
-+------+-------------------------------------------------------+
-| N817 | camelcase imported as acronym                         |
-+------+-------------------------------------------------------+
++---------+---------------------------------------------------------+
+| code    | sample message                                          |
++=========+=========================================================+
+| _`N801` | class names should use CapWords convention              |
+|         | (https://pep8.org/#class-names)                         |
```

Comment:
Please use the real pep link. I don't want to drive traffic to someone who raised over 30k for a project and then ran away with the funds.

Reply:
I'll change them. I started with python.org URLs, but they're much longer, and reST makes you inflate the table to the maximum cell's width. Maybe I'll avoid that using hyperlink aliases or something.

Reply:
You can do things like:

```rst
`class names should use CapWords convention <class-names>`_
```

And later in the file do


```rst
.. _class-names: https://python.org...
```